src/server/navigation.ts

```diff
@@ -175,12 +199,24 @@ function getSearchPath(node: Node): Search[] {
             {
                 type: 'variable',
                 name,
+                start: spanToPos(
+                    node.end
+                        ? [node.end[0], node.end[1] - name.length]
+                        : undefined,
+                ),
+                end: spanToPos(node.end),
```

Comment:
Is this to try to find the range of the variable? I have added them in this commit: https://github.com/serokell/motoko/pull/50/commits/870dc6e708b74786da23cfdcc0509b16a2e1a93f

Reply:
I think I will cherry-pick your commit into separate branch.

Reply:
reworked